_posts/2017-01-27-squad-health-checks.md

```diff
@@ -0,0 +1,70 @@
+---
+layout:     post
+title:      Squad Health Checks
+date:       2017-01-27 10:00
+summary:    Join Adele, Mark, Tom, Mark, Adam & John for our January 2017 edition of Sky Bet Tech Talks on Retrospectives, AGILE Programme Management, Discovery & Experimentation and Squad Health Checks
+category:   Tech Talks
+tags:       Tech, Talk, Agile, Squads, Health Checks
+author:     john_kitson
+---
+
+As a scrum master or agile coach, it's really important to enable a culture of continuous improvement. One way of doing this is to get the teams you work with into the habit of retrospecting. Ideally teams are always on the look-out for ways to improve their capability to meet business needs both now and in the future. Having heartbeat meetings; retrospectives where the team can discuss problems and hindrances, and ways to optimise their approach is a key part of agile working.
```

Comment:
> Having heartbeat meetings; _retrospectives where the team can discuss problems and hindrances, and ways to optimise their approach_

the bit is italics is parenthetical thought and should be delimited by em dashes or parens rather than a leading semicolon.

> Having heartbeat meetings — retrospectives where the team can discuss problems and hindrances, and ways to optimise their approach  — is a key part of agile working.

or even less cumbersomely, use a colon to make it clear as a definition:
> A key part of agile working is having heartbeat meetings: retrospectives where the team can discuss problems and hindrances, and ways to optimise their approach